patchwork-events-entity/src/main/java/net/minecraftforge/event/entity/player/PlayerEvent.java

```diff
@@ -73,8 +82,7 @@ public PlayerEntity getPlayer() {
 	Visibility
 	ItemPickupEvent
 	ItemCraftedEvent
-	ItemSmeltedEvent
-	PlayerLoggedInEvent
+	ItemSmeltedEventLogg
```

Comment:
Huh?

Reply:
IJ copys things around when you drag with your mouse sometimes.